src/saving/SaveHelper.cs

```diff
@@ -131,11 +129,48 @@ private static Save CreateSaveObject(MainGameState gameState, SaveInformation.Sa
         };
     }
 
+    private static string BuildQuicksaveFilename(uint quicksaveId)
+        => $"quick_save.{quicksaveId}.{Constants.SAVE_EXTENSION}";
+
+    /// <summary>
+    ///   Goes through the save-folder to detect the last quicksave-id
+    /// </summary>
+    private static uint GetLastQuicksaveId()
+    {
+        using (var directory = new Directory())
+        {
+            directory.Open(Constants.SAVE_FOLDER);
+            for (uint i = 0; ; i++)
+            {
+                if (!directory.FileExists(BuildQuicksaveFilename(i)))
+                    return i - 1;
+            }
+        }
+
+        /*var file = new File();
+
+        uint quicksaveId;
+
+        // if the file exists: read int and add one to it.
+        // if it doesn't: create the file with savegameId 0.
+        if (file.FileExists(filePath))
+        {
+            file.Open(filePath, File.ModeFlags.Read);
+            quicksaveId = file.Get32();
+            file.Close();
+        }
+        else
+        {
+            quicksaveId = 0;
+        }
+
+        return quicksaveId;*/
+    }
+
     private static void PerformSave(Save save, SaveInformation.SaveType type, Stopwatch stopwatch)
     {
         // TODO: implement type naming
-        var name = "quick_save." + Constants.SAVE_EXTENSION;
-
+        var name = BuildQuicksaveFilename(GetLastQuicksaveId() + 1);
```

Comment:
The quick save numbers should be reused so that after quick save `quick_save_5` the first quick save should be overwritten.